### PR TITLE
MAUI-3721 : Add UG documents for .NET MAUI backdrop page control

### PR DESCRIPTION
**Pull request by BalachandarSelvam**
Hi @vigneshrameshsync,

I have added UG document for the backdrop page control.

Regards, 
Balachandar.

### CODE REVIEW

File: MAUI/Backdrop/reveal-height-customization.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+documentation: ug
+---
+
+# Reveal Height Customization .NET MAUI Backdrop Page (SfBackdropPage)
+
+When revealing the `back layer`, the front layer will be moved downwards. By setting the `BackLayerRevealOption` property of backdrop, you can customize how far the front layer can be moved from the header when revealing the back layer.
+
+The following options are provided to move the front layer:
+
+`Fill` – Moves the front layer downwards to the entire height of the page excluding the `RevealedHeight`.
```
Comment (Chrispineagunja) [inline MAUI/Backdrop/reveal-height-customization.md:16]: `Fill` – Moves the front layer downwards to the entire height of the page, excluding the `RevealedHeight.`

File: MAUI/Backdrop/Events.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+platform: MAUI
+control: SfBackdropPage
+documentation: ug
+---
+
+# Events in .NET MAUI Backdrop Page (SfBackdropPage)
+
+## BackLayerStateChanged event
+
+The `BackLayerStateChanged` event occurs when the backdrop page back layer is revealed and concealed. The event occurs in the cases mentioned in this `documentation`.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/Events.md:14]: made changes

File: MAUI/Backdrop/corner-shape-customization.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+title: Corner shape customization of Syncfusion Backdrop Page | Syncfusion
+description: This section explains how to customize the corners of the front layer of the backdrop to flat and curve shapes. 
+platform: MAUI
+control: SfBackdropPage
+documentation: ug
+---
+
+# Corner Shape Customization in .NET MAUI Backdrop Page (SfBackdropPage)
+
+Backdrop allows to customize the shapes on the top corners of the front layer. Curved and cut (flat) shape options are available , it can be switched by setting `EdgeShape` property of the front layer.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/corner-shape-customization.md:12]: made changes

File: MAUI/Backdrop/corner-shape-customization.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+![.NET MAUI Backdrop CornerType image](images/corner-shape/edge-shape.png)
+
+N> The backdrop can only be shaped on the top left and top right corners.
+
+N> If you need to set background color for the back layer, set the `BackgroundColor` property to the content of the BackdropBackLayer instead of BackdropBackLayer itself.
+
+Both the side of the corner radius can be customized separately by setting `LeftCornerRadius` and `RightCornerRadius` properties of `BackdropFrontLayer`.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/corner-shape-customization.md:20]: made changes

File: MAUI/Backdrop/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+title: Getting started with .NET MAUI Backdrop Page| Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI Backdrop Page(SfBackdropPage) control, its elements and more.
+platform: MAUI
+control: SfBackdropPage
+documentation: ug
+---
+
+# Getting started with .NET MAUI Backdrop Page
+
+This section explains the steps required to add the Backdrop page and its elements such as BackLayer, FrontLayer in project. This section covers only the basic features needed to know and gets started with the Syncfusion backdrop page.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/getting-started.md:12]: made changes

File: MAUI/Backdrop/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+
+## Creating an application using the .NET MAUI Backdrop Page
+
+* Create a new .NET MAUI application in the Visual Studio.
+
+* The Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add SfBackdropPage to your project, open the NuGet package manager in Visual Studio, and search for [Syncfusion.Maui.Backdrop] then install that.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/getting-started.md:18]: made changes

File: MAUI/Backdrop/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+N> `Title` and `ToolBarItems` properties of the `Page` can used to customize the appearance of header.
+
+### Configure header
+
+Page header for the backdrop appears only when adding backdrop as a child of NavigationPage. To know more about it, refer to `header configuration`.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/getting-started.md:96]: made changes

File: MAUI/Backdrop/getting-started.md [hunk 8]
```diff
@@ -0,0 +1,225 @@
...
+    }
+};
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Reveal and conceal the back layer
+
+The following options are provided in backdrop to reveal and conceal the back layer:
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/getting-started.md:183]: made changes

File: MAUI/Backdrop/getting-started.md [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+
+* **Programmatically** - Reveals back layer by setting the `IsBackLayerRevealed` property to true. By default, it is set to false.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/getting-started.md:185]: made changes

File: MAUI/Backdrop/getting-started.md [hunk 10]
```diff
@@ -0,0 +1,225 @@
...
+* **Touch interaction** - Reveals back layer by clicking the tool bar icon at the top-right corner of navigation bar header. The Hamburger (or menu ) icon reveals and the Close icon conceals the back layer. When adding the backdrop as a child of FlyoutPage, the Hamburger and Close icons will be replaced by expand (or down arrow) and collapse (or up arrow) icons, respectively.
+* **Swipe or fling action** - Reveals back layer by swipe/fling action on the front layer to the required direction. Swipe downwards to reveal, and swipe upwards to conceal the back layer.  The swipe/fling action will be handled only on the top of the front layer to the `RevealedHeight`.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/getting-started.md:187]: made changes

File: MAUI/Backdrop/getting-started.md [hunk 11]
```diff
@@ -0,0 +1,225 @@
...
+
+To know more information about reveal height customization ,refer this `link`.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/getting-started.md:189]: made changes

File: MAUI/Backdrop/getting-started.md [hunk 12]
```diff
@@ -0,0 +1,225 @@
...
+
+#endregion
+
+{% endhighlight %}
+
+{% endtabs %}
+
+N> 
+* Get the complete getting started sample from GitHub `link.`
+* Refer to our `.NET MAUI Backdrop` feature tour page for its groundbreaking feature representations.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/getting-started.md:225]: made changes

File: MAUI/Backdrop/header-configuration.md [hunk 13]
```diff
@@ -0,0 +1,130 @@
...
+title: Header Configuration with Syncfusion Backdrop Page | Syncfusion
+description: Learn here all about how to configure and customize the header in the Syncfusion backdrop page control in .NET MAUI platform.
+platform: MAUI
+control: SfBackdropPage
+documentation: ug
+---
+
+# Header Configuration
+
+Add backdrop page as a children of [`NavigationPage`](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/pages/navigationpage?view=net-maui-6.0) in App.xaml.cs class. Also, `BarBackgroundColor`, `BarTextColor` and other properties of [`NavigationPage`](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/pages/navigationpage?view=net-maui-6.0) can be set to customize the default appearance of header.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/header-configuration.md:12]: made changes

File: MAUI/Backdrop/header-configuration.md [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
+   … 
+  MainPage = new NavigationPage(new BackdropSamplePage());
+   … 
+}
+
+#endregion 
+
+{% endhighlight %}
+
+N> Page header for the backdrop will appear only when adding backdrop as a children of [`NavigationPage`](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/pages/navigationpage?view=net-maui-6.0).
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/header-configuration.md:30]: made changes

File: MAUI/Backdrop/header-configuration.md [hunk 15]
```diff
@@ -0,0 +1,130 @@
...
+ 
+## Icon customization
+
+The default icons in the navigation header can be customized using the following ways:
+
+### Default icons in NavigationPage
+
+When the backdrop page contained within the [NavigationPage](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/pages/navigationpage?view=net-maui-6.0) , hamburger icon and close icon (X mark ) will be used by default.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/header-configuration.md:38]: made changes

File: MAUI/Backdrop/header-configuration.md [hunk 16]
```diff
@@ -0,0 +1,130 @@
...
+
+![.NET MAUI Backdrop in navigation page](images/header-configuration/default-navigationpage-icons.png)
+
+### Default icons in FlyoutPage
+
+When the backdrop page placed in the [FlyoutPage](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/pages/flyoutpage?view=net-maui-6.0), down arrow icon and up arrow icon will be used by default.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/header-configuration.md:44]: made changes

File: MAUI/Backdrop/header-configuration.md [hunk 17]
```diff
@@ -0,0 +1,130 @@
...
+
+![.NET MAUI Backdrop in navigation page](images/header-configuration/default-flyoutpage-icons.png)
+
+### Custom icons
+
+Customize the default icons in the navigation header by setting the `OpenIconImageSource` and `CloseIconImageSource` properties in `SfBackdropPage`. It accepts following inputs:
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/header-configuration.md:50]: made changes

File: MAUI/Backdrop/header-configuration.md [hunk 18]
```diff
@@ -0,0 +1,130 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Backdrop Custom icons](images/header-configuration/custom-icons.png)
+ 
+## Icon text customization
+
+You can customize the icon text in the navigation header by setting the `OpenText` and `CloseText` properties in `SfBackdropPage`.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/header-configuration.md:95]: made changes

File: MAUI/Backdrop/migration.md [hunk 19]
```diff
@@ -0,0 +1,273 @@
+---
+layout: post
+title: Migrate from Xamarin to .NET MAUI  Backdrop page | Syncfusion
+description: Learn here all about Migrating from Syncfusion Xamarin backdrop page to Syncfusion .NET MAUI backdrop page control and more.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/migration.md:4]: made changes

File: MAUI/Backdrop/migration.md [hunk 20]
```diff
@@ -0,0 +1,273 @@
...
+platform: MAUI
+control: SfBackdropPage
+documentation: ug
+---  
+
+# Migrate from Xamarin.Forms SfBackdropPage to .NET MAUI SfBackdropPage
+
+To make the migration from the [Xamarin SfBackdropPage](https://www.syncfusion.com/xamarin-ui-controls/xamarin-backdrop) to `.NET MAUI SfBackdropPage` easier, most of the APIs from the Xamarin SfBackdropPage were kept in the .NET MAUI SfBackdropPage. However, to maintain the consistency of API naming in the .NET MAUI SfBackdropPage, some of the APIs have been renamed. Please find the difference in the following topics.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/migration.md:12]: made changes

File: MAUI/Backdrop/migration.md [hunk 21]
```diff
@@ -0,0 +1,273 @@
...
+{% endhighlight %}
+
+{% endtabs %}
+</td>
+</tr>
+</table>
+
+## Unsupported features from Xamarin.Forms
+
+* [`OpenIcon`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Backdrop.SfBackdropPage.html#Syncfusion_XForms_Backdrop_SfBackdropPage_OpenIcon) and [`CloseIcon`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Backdrop.SfBackdropPage.html#Syncfusion_XForms_Backdrop_SfBackdropPage_CloseIcon) support has not been provided in the SfBackdropPage. Instead, you can achieve it using the `OpenIconImageSource` and `CloseIconImageSource` property. Since the [`OpenIcon`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Backdrop.SfBackdropPage.html#Syncfusion_XForms_Backdrop_SfBackdropPage_OpenIcon) and [`CloseIcon`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Backdrop.SfBackdropPage.html#Syncfusion_XForms_Backdrop_SfBackdropPage_CloseIcon) are marked obsolete.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/migration.md:273]: made changes

File: MAUI/Backdrop/overview.md [hunk 22]
```diff
@@ -0,0 +1,26 @@
...
+title: What's New in Essential Studio for .NET MAUI | Syncfusion
+description: A .NET MAUI backdrop appears behind all the surfaces in an app, displaying contextual and actionable content.
+platform: MAUI
+control: SfBackdropPage
+documentation: ug
+---
+
+# .NET MAUI Backdrop Page (SfBackdropPage) Overview
+
+The .NET MAUI Backdrop page is composed of two surfaces, a back layer and a front layer. The back layer holds actionable content (like navigation or filtration), which is relevant to the front layer.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/overview.md:12]: made changes

File: MAUI/Backdrop/overview.md [hunk 23]
```diff
@@ -0,0 +1,26 @@
...
+
+### Key features
+
+ * Allows to add with NavigationPage and supports seamless navigation and toolbar customizations.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/overview.md:16]: made changes

File: MAUI/Backdrop/overview.md [hunk 24]
```diff
@@ -0,0 +1,26 @@
...
+ 
+ * Adjusts the height of back layer automatically based on its content and provides an option to expand the content to fit the screen.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/overview.md:18]: made changes

File: MAUI/Backdrop/overview.md [hunk 25]
```diff
@@ -0,0 +1,26 @@
...
+ 
+ * Supports curved and flat edge shapes for the front layer with corner radius customization options.
+ 
+ * Provides smooth animations for revealing and concealing the back layer content.
+ 
+ * Allows users to customize the icons of navigation header in the revealed and closed states separately.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/overview.md:24]: made changes

File: MAUI/Backdrop/reveal-height-customization.md [hunk 26]
```diff
@@ -0,0 +1,83 @@
...
+title: Reveal height customization of Syncfusion Backdrop Page | Syncfusion
+description: Learn here all about how to configure and customize the reveal height in the Syncfusion backdrop page control.
+platform: MAUI
+control: SfBackdropPage
+documentation: ug
+---
+
+# Reveal Height Customization .NET MAUI Backdrop Page (SfBackdropPage)
+
+When revealing the `back layer`, the front layer will be moved downwards. By setting the `BackLayerRevealOption` property of backdrop, you can customize how far the front layer can be moved from the header when revealing the back layer.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/reveal-height-customization.md:12]: made changes

File: MAUI/Backdrop/reveal-height-customization.md (hunk 1 above)
Comment (BalachandarSelvam) [reply MAUI/Backdrop/reveal-height-customization.md:16]: made changes

File: MAUI/Backdrop/getting-started.md [hunk 27]
```diff
@@ -0,0 +1,225 @@
...
+{% endhighlight %}
+
+{% endtabs %}
+
+## Reveal and conceal the back layer
+
+The following options are provided in backdrop to reveal and conceal the back layer:
+
+* **Programmatically** - Reveals back layer by setting the `IsBackLayerRevealed` property to true. By default, it is set to false.
+* **Touch interaction** - Reveals back layer by clicking the tool bar icon at the top-right corner of navigation bar header. The Hamburger (or menu ) icon reveals and the Close icon conceals the back layer. When adding the backdrop as a child of FlyoutPage, the Hamburger and Close icons will be replaced by expand (or down arrow) and collapse (or up arrow) icons, respectively.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/Backdrop/getting-started.md]: Made changes except - "the Hamburger and Close icons will be replaced by expanding (or down arrow) and collapsing (or up arrow) icons respectively "

**Review by vigneshrameshsync — APPROVED** [review]
Fine.

**Pull request merged**